src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -244,36 +280,35 @@ SourceInfo<string> GetTopicHref(TableOfContentsItem tocInputModel)
                 return default;
             }
 
-            (SourceInfo<string> resolvedTocHref, TableOfContentsItem resolvedTopicItem, TableOfContentsModel subChildren) ProcessTocHref(SourceInfo<string> tocHref)
+            (SourceInfo<string> resolvedTocHref, TableOfContentsModel subChildren, (TableOfContentsItem item, Document doc) firstChild, TocHrefType tocHrefType) ProcessTocHref(SourceInfo<string> tocHref)
```

Comment:
>(TableOfContentsItem item, Document doc)  [](start = 83, length = 41)

From the caller, seems it is not necessary to make return type a nested tuple.

Reply:
it's necessary, firstItem is used separated